scripts/eslint_rules/no-deprecated-clone.js

```diff
@@ -10,30 +10,65 @@ function getVariableDefinition(name /*: string */, scope /*: Scope */) {
   } while ((currentScope = currentScope.upper));
 }
 
-function isImportedFrom(
+function isImportedFromTypes(
+  context /*: Context */,
   node /*: Node */,
-  scope /*: Scope */,
-  module /*: string */,
   kind /*: "named" | "namespace" */
 ) {
-  const definition = getVariableDefinition(node.name, scope);
+  const definition = getVariableDefinition(node.name, context.getScope());
+
   if (!definition) return false;
+  if (definition.type !== "ImportBinding") return false;
+  if (definition.parent.source.value !== "@@babel/types") return false;
```

Comment:
double `@`?

Reply:
I was also wondering, I assumed it was some sort of escape.